projects/1.18/assets/little-contraptions/littlecontraptions/lang/zh_cn.json

```
@@ -0,0 +1,44 @@
+{
+  "item.littlecontraptions.contraption_barge": "动态结构驳船",
+  "block.littlecontraptions.barge_assembler": "驳船装配站",
+  
+  "littlecontraptions.ponder.tag.tugs": "Little Logistics 拖船",
+  "littlecontraptions.ponder.tag.tugs.description": "自带寻路的水栖火车！",
+  "littlecontraptions.ponder.tag.trains": "Little Logistics 机车",
+  "littlecontraptions.ponder.tag.trains.description": "麻雀虽小，五脏俱全！",
+
+  "littlecontraptions.ponder.basic_assembler.header": "使用驳船装配站控制结构移动",
+  "littlecontraptions.ponder.basic_assembler.text_1": "驳船装配站会将它所连接的结构安装到动态结构驳船上",
+  "littlecontraptions.ponder.basic_assembler.text_2": "若没有通入红石信号，它会将驳船装置解散为方块的形式",
+  "littlecontraptions.ponder.basic_assembler.text_3": "与列车装配站类似，你可以用底盘和强力胶将大型动态结构安装上去",
+
+  "littlecontraptions.ponder.basic_tug.header": "拖船的基础知识",
+  "littlecontraptions.ponder.basic_tug.text_1": "拖船可放置在任何水域中",
+  "littlecontraptions.ponder.basic_tug.text_2": "它们会根据自己装载的航线配置器自动寻找路径",
+  "littlecontraptions.ponder.basic_tug.text_3": "把配置器放进拖船，即可让其起锚航行！",
+  "littlecontraptions.ponder.basic_tug.text_4": "有时拖船会在转弯处卡住。转角导轨能解决这个问题",
+  "littlecontraptions.ponder.basic_tug.text_5": "如果转角导轨的方向不正确，可用列车长扳手改变方向",
+  
+  "littlecontraptions.ponder.tug_dock.header": "使拖船停靠码头",
+  "littlecontraptions.ponder.tug_dock.text_1": "码头会在拖船装载/卸载时使其停止航行",
+  "littlecontraptions.ponder.tug_dock.text_2": "所有码头都需要有一个拖船码头和排成直线的一排驳船码头",
```

```suggestion
  "littlecontraptions.ponder.tug_dock.text_2": "所有码头都需要有一个拖船码头方块和排成直线的一排驳船码头方块",
```
码头需要码头？
原文：**Every docking station needs to have one tug dock block and a direct line of barge docks blocks**
| 严重程度         | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |  歧义  |